source/_patterns/01-atoms/dropdown/demo/dropdowns.twig

```diff
@@ -0,0 +1,201 @@
+<div class="container">
+  {% embed '@atoms/grid/_grid-1-up.twig' with { container: true } %}
+    {% block column_1 %}
+
+      <h1>Dropdowns</h1>
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_text: dropdowns_helper,
+        card_background: 'secondary',
```

Comment:
Most of these dropdowns use the same color, find a way to use that color that is not hand-coding like this nor setting it as the default color because that causes problems elsewhere

Reply:
I don't mind manually setting. The work is done and it provides a good example for how to do something by looking at code. FWIW.